homework/content.js

```diff
@@ -4,29 +4,31 @@
   var _wrapper = document.querySelector('#note-content-wrapper');
 
   function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
+    var con = new content();
+    window.addEventListener('note-open', 
```

Comment:
You should bind the listener in argument list with `this`, rather than an anonymous function. See the second example on the MDN page:

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Why_use_addEventListener